pkg/kube/snapshot/snapshot_alpha.go

```diff
@@ -0,0 +1,317 @@
+// Copyright 2019 The Kanister Authors.
```

Comment:
for new files, please use 2020

```suggestion
// Copyright 2020 The Kanister Authors.
```